src/lavinmq/http/controller/prometheus.cr

```diff
@@ -252,6 +252,16 @@ module LavinMQ
                       value: @amqp_server.stats_system_collection_duration_seconds.to_f,
                       type:  "gauge",
                       help:  "Time it takes to collect system metrics"})
+        writer.write({name:  "total_connected_followers",
+                      value: @amqp_server.@replicator.followers.size,
+                      type:  "gauge",
+                      help:  "Amount of follower nodes connected"})
+        @amqp_server.@replicator.followers.each_with_index do |f, i|
+          writer.write({name:  "follower_#{i}_lag",
```

Comment:
Could we write something more specific about the follower here so they can be identified? "follower_1" doesn't really mean anything IMO, so would be better with hostname or IP or something.

Reply:
> Could we write something more specific about the follower here so they can be identified? "follower_1" doesn't really mean anything IMO, so would be better with hostname or IP or something.

I added remote ip as an id on the name. Might get a little long, and harder to read? 

```
# TYPE lavinmq_total_connected_followers gauge
# HELP lavinmq_total_connected_followers Amount of follower nodes connected
lavinmq_total_connected_followers 2
# TYPE lavinmq_follower_lag_127.0.0.1:5679 gauge
# HELP lavinmq_follower_lag_127.0.0.1:5679 Lag for follower on address: 127.0.0.1:5679
lavinmq_follower_lag_127.0.0.1:5679 0
# TYPE lavinmq_follower_lag_127.0.0.1:5679 gauge
# HELP lavinmq_follower_lag_127.0.0.1:5679 Lag for follower on address: 127.0.0.1:5679
lavinmq_follower_lag_127.0.0.1:5679 0
```